### ISSUE TITLE
No full config list in README

### ISSUE THREAD

**Issue by StevenUpForever** [TestArmada/flank#340]
There's all config properties list and command examples before in `README` and now under `flank_java` branch, but not under `master` branch, need command line examples and config list in `README`.

**Comment by bootstraponline** [comment]
![image](https://user-images.githubusercontent.com/1173057/46702162-19308580-cbf0-11e8-9072-719c049bc8aa.png)
![image](https://user-images.githubusercontent.com/1173057/46702170-20579380-cbf0-11e8-9bc0-5b6e887a282b.png)

The master branch readme contains a full listing of all options.

**Comment by StevenUpForever** [comment]
Ok, could you help confirm a few things?
1. Any old options that not listed, e.g. `debug-prints` or name changed means either it's no longer supported or new version applies to new option names?
1. On Android side, will it support `config.properties` now?
1. Any command line arguments support as before? e.g. https://github.com/TestArmada/flank/tree/flank_java#run-tests

**Comment by bootstraponline** [comment]
> 3\. Any command line arguments support as before? e.g.

Most command line arguments have been removed in favor of a yaml based file format.

> * Any old options that not listed, e.g. `debug-prints` or name changed means either it's no longer supported or new version applies to new option names?
> * On Android side, does it support `config.properties` now?

We standardized on the official gcloud yaml. This means the same config works with both flank and gcloud:

- `gcloud firebase test android run flank.yml:gcloud`
- `flank firebase test android run`

If gcloud supports an option and Flank doesn't then that's typically a feature we'd want to add.
- https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run

The legacy options from Flank Java (`debug-prints`) are not supported. The code base has completely been rewritten in Kotlin + APIs instead of Java + gcloud.

There's a #flank channel on https://firebase.community if you have more general usage questions.
